src/backend/distributed/ddl/foreign_constraint.c

```diff
@@ -649,3 +649,63 @@ HeapTupleOfForeignConstraintIncludesColumn(HeapTuple heapTuple, Oid relationId,
 
 	return false;
 }
+
+
+/*
+ * This function will be removed later
+ */
+List *
+GetReferencedReferenceTableList(Oid relationId)
```

Comment:
you can rebase your changes to `foreign_key_relations ` to get rid of this function and rely on 
```C
		DistTableCacheEntry *cacheEntry = DistributedTableCacheEntry(relationId);
		List *referencedRelations = cacheEntry->referencedRelationsViaForeignKey;

```